### DISCUSSION TITLE
Support for custom connectors

### DISCUSSION THREAD

**Discussion by klzbrt** [gsimardnet/PowerQueryNet#18] [Ideas]
Great work, Guillaume! I am interested in the ability to add support for custom Power BI Connectors. We have a connector ([Vena)](https://docs.microsoft.com/en-us/power-query/connectors/#letter-v) that we'd like to use for automating data pulls using m.

**Comment by gsimardnet** [comment]
Hi klzbrt,

You have a very good suggestion. So far I didn't look for that kind of support but I will try to do some research on the requirements in the next couple of days.

I will answer my findings in discussion #19 .
